packaging/common/khiops/WHATSNEW.txt

```diff
@@ -2,9 +2,17 @@
 Release Notes for Khiops 10 Version Series
 ==========================================
 
-Version 10.2.3
+Version 10.2.4
 ==============
+Bug fix:
+- Fix MPI executable detection in "Conda-based" environments on Linux and MacOS.
+
+Minor improvements:
+- Reduce launching time of the GUI on Windows.
+- Let Conda choose the build tools for building the Windows Conda packages.
```

Comment:
Je ne pense pas que ce deuxième "minor improvement" (Let conda choose...) soit pertinent pour les utilisateurs.
Je pense qu'on pourrait le supprimer (en passant "Minor improvement" au singulier)

Reply:
Ok, fair enough. I will remove this second improvement.

Reply:
Done.